### DISCUSSION TITLE
Laravel documentation back to top button

### DISCUSSION THREAD

**Discussion by sts-ryan-holton** [laravel/framework#48331] [Ideas]
Hi, any chance to get a "back to top" button to scroll the user back to the top of the documentation? Often find myself reloading the page!

**Comment by studnitz** [comment]
You can use <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>⬆</kbd> or just the <kbd>Home</kbd> key for that.

**Reply by Angel5a** [reply]
...Just attach keyboard to your phone. ;)

**Reply by YasserB94** [reply]
@Angel5a 

**iOS users - Safari**
😌Just tap inside of the 'safe zone' on the top of your screen (On the battery🔋, time🕰️, signal indicators🛜,...)

_Navigation Bar on top of the screen ?_
To activate this behaviour you need the UI to be active do one of these to expand the UI:
- Scroll up a little bit
- Tap the navigation bar

🏆BONUS TIP: 
Annoyed your thumb has to stretch the whole screen for this ? 'flick' the bottom bar down with your thumb to bring the top part of the screen down 50% of your phone. 

**Android users**
🛠️🤖🔧
I cannot provide you with simple steps.
Android users love to tell me how customisable their OS is, so I'm quite sure I don't need to either.
